### ISSUE TITLE
Helper to add fairlearn's MetricFrame to the model card

### ISSUE THREAD

**Issue by adrinjalali** [skops-dev/skops#278]
One can easily disaggregate metrics by sensitive attribute or arbitrary groups using fairlearn's [`MetricFrame`](https://fairlearn.org/v0.8/api_reference/fairlearn.metrics.html#fairlearn.metrics.MetricFrame). It'd be nice for a user to be able to easily add that to a model card.

Example usage of a MetricFrame looks like:

```py
from fairlearn.metrics import MetricFrame, selection_rate
from sklearn.metrics import accuracy_score
import pandas as pd
y_true = [1,1,1,1,1,0,0,1,1,0]
y_pred = [0,1,1,1,1,0,0,0,1,1]
sex = ['Female']*5 + ['Male']*5
metrics = {"selection_rate": selection_rate}
mf1 = MetricFrame(
     metrics=metrics,
     y_true=y_true,
     y_pred=y_pred,
     sensitive_features=sex)
```
The reported metric can be any sklearn score or a fairlearn metric.

**Comment by BenjaminBossan** [comment]
That would be good to have.

How would it look like? IIUC, the metric frame would have a couple of attributes/methods that return a table. We could add all those tables separately using `.add_table`. We could also pivot those tables so that they only have two columns (the name of the metric and the value) and then add the resulting rows via `add_metrics`.

**Comment by merveenoyan** [comment]
TIL of `MetricFrame`, indeed looks very good 🤩 I agree, could be handled with what we have already and we could just release an example of this. (like we did with ELI5) WDYT?

**Comment by lazarust** [comment]
I'd like to work on this if it hasn't already been started.

**Comment by BenjaminBossan** [comment]
@lazarust I think no one has started on this yet, so you can take it.
